### PR TITLE
Addition of 5 new questions (in a comment right now)

### PR DESCRIPTION
**Pull request by orenpinto**
Update of performance metric, shows the amount of questions played till now instead of total questions amount

### CODE REVIEW

File: app/src/main/java/com/example/palabra/QuestionsDB.java [hunk 1]
```diff
@@ -45,13 +45,53 @@
...
+//        sixthQuestion.add("Beach");
+//        sixthQuestion.add("Playa");
+//        sixthQuestion.add("Biblioteca");
+//        sixthQuestion.add("Mercado");
+//        sixthQuestion.add("Piscina");
+//        questionsList.add(sixthQuestion);
+//
+//        ArrayList<String> seventhQuestion = new ArrayList<>();
+//        seventhQuestion.add("Bathroom");
+//        seventhQuestion.add("Baño");
```
Comment (sapirshp) [inline app/src/main/java/com/example/palabra/QuestionsDB.java]: write the correct answer without special characters so I culd match the image name to it

File: app/src/main/java/com/example/palabra/QuestionsDB.java [hunk 2]
```diff
@@ -45,13 +45,53 @@
...
+//        eighthQuestion.add("Eye");
+//        eighthQuestion.add("Ojo");
+//        eighthQuestion.add("Cabeza");
+//        eighthQuestion.add("Oreja");
+//        eighthQuestion.add("Naríz");
+//        questionsList.add(eighthQuestion);
+//
+//        ArrayList<String> ninthQuestion = new ArrayList<>();
+//        ninthQuestion.add("Lightning");
+//        ninthQuestion.add("Relámpago");
```
Comment (sapirshp) [inline app/src/main/java/com/example/palabra/QuestionsDB.java]: same here about the special characters

File: app/src/main/java/com/example/palabra/QuestionsDB.java (hunk 1 above)
Comment (orenpinto) [reply app/src/main/java/com/example/palabra/QuestionsDB.java]: fixed

File: app/src/main/java/com/example/palabra/QuestionsDB.java (hunk 2 above)
Comment (orenpinto) [reply app/src/main/java/com/example/palabra/QuestionsDB.java]: fixed

**Pull request merged**